src/main/features/claimant-response/routes/counter-offer-accepted.ts

```diff
@@ -34,10 +34,16 @@ export default express.Router()
       const isCourtOrderPaymentPlanConvertedByDefendantFrequency =
         (defendantPaymentOption !== 'BY_SPECIFIED_DATE') ? defendantPaymentPlan.frequency && differentPaymentFrequency : false
 
+      let courtOrderPaymentPlan
+
+      if (draft.document.courtOfferedPaymentIntention !== undefined) {
```

Comment:
Could still be `null`.
Suggest either `if (draft.document.courtOfferedPaymentIntention) {` or just using the ternary boolean operator for the options `courtOrderPaymentPlan: draft.document.courtOfferedPaymentIntention ? draft.document.courtOfferedPaymentIntention.repaymentPlan : undefined`

Reply:
Yeah ok